### PR TITLE
Expose tempvars2 on the runtime & minor code changes.

### PR DESCRIPTION
**Pull request by yuri-kiss**
Please read the comments I put on the git commit for why stuff was changed.

As per what cst1229 did, for letting others have access to the extension stuff then lets us access the previously accessible runtime variables from another extension.

And the rest is just some minor code changes.

### CODE REVIEW

File: extensions/Lily/TempVariables2.js [hunk 1]
```diff
@@ -7,28 +7,28 @@
...
     hideFromPalette: hidden,
   });
 
-  function resetRuntimeVariables() {
-    runtimeVariables = Object.create(null);
-  }
-
   class TempVars {
     constructor() {
+      // this.resetRntimeVariables would be preferable but,
```
Comment (LilyMakesThings) [inline extensions/Lily/TempVariables2.js:19]: typo: this.resetRuntimeVariables

**Review by LilyMakesThings — APPROVED** [review]

**Comment by CubesterYT** [comment]
I'll look into this more clearly later, but it's looking fine as of now.

File: extensions/Lily/TempVariables2.js (hunk 1 above)
Comment (yuri-kiss) [reply extensions/Lily/TempVariables2.js]: oop will fix lol

**Review by LilyMakesThings — APPROVED** [review]

**Review by SharkPool-SP — APPROVED** [review]

**Review by CubesterYT — APPROVED** [review]
I say this is a certified anti YandereDev moment

**Pull request merged**